src/main/java/com/commercetools/project/sync/Syncer.java

```diff
@@ -72,32 +74,42 @@ public Syncer(
    * @return completion stage containing no result after the execution of the sync process and
    *     logging the result.
    */
-  CompletionStage<Void> sync() {
-    LOGGER.info("Starting sync..");
+  public CompletionStage<Void> sync() {
+
+    if (LOGGER.isInfoEnabled()) {
+      LOGGER.info(format("Starting %s..", sync.getClass().getSimpleName()));
+    }
+
     return queryAll(sourceClient, query, this::syncPage)
         .thenAccept(
             ignoredResult -> {
               logStatistics(sync.getStatistics(), LOGGER);
               final String successMessage =
                   format(
-                      "Syncing from CTP project '%s' to project '%s' is done.%n",
+                      "%nSyncing %s from CTP project with key '%s' to project with key '%s' is done.%n",
```

Comment:
not sure exactly what do you mean here? but the point here is to make it clear when running all the syncers for example, then the user would get to see sth like this:

````
"Syncing Products from CTP project with key 'foo' to project with key 'bar' is done."
......
"Syncing Categories from CTP project with key 'foo' to project with key 'bar' is done."
......
"Syncing ProductTypes from CTP project with key 'foo' to project with key 'bar' is done."
````

instead of seeing: 

````
"Syncing from CTP project with key 'foo' to project with key 'bar' is done."
......
"Syncing from CTP project with key 'foo' to project with key 'bar' is done."
......
"Syncing from CTP project with key 'foo' to project with key 'bar' is done."
````

the second one your are not sure which sync process is done. 

I am not sure if I answered your question though.

Reply:
Let me explain with code, but you can skip these comments because both of them doing the same thing.

instead of using getResourceName, 
in Syncer.java I added the code below:
``` java
protected abstract String getSuccessMessage();
```

now, every implementation of Syncer has to implement it like this.

``` java
@Override
  protected String getSuccessMessage() {
    return format("%nSyncing Categories from CTP project with key '%s' to project with key '%s' is done.%n",
                sourceClient.getConfig().getProjectKey(),
                targetClient.getConfig().getProjectKey());
  }
```

``` java
public CompletionStage<Void> sync() {

    if (LOGGER.isInfoEnabled()) {
      LOGGER.info(format("Starting %s..", sync.getClass().getSimpleName()));
    }

    return queryAll(sourceClient, query, this::syncPage)
        .thenAccept(
            ignoredResult -> {
              logStatistics(sync.getStatistics(), LOGGER);
              System.out.println(getSuccessMessage()); // NOPMD
            });
  }
```

Reply:
aha! Nice idea :)

Reply:
I removed this log message at all. Since the summary for each sync process was enough: 
378471a

Reply:
less is better 👍